### PR TITLE
 changes button color and some and font bold with feedback enhancment

### PR DESCRIPTION
**Pull request by NoorAlkattan**

### CODE REVIEW

File: app/views/businesses/registrations/edit.html.haml [hunk 1]
```diff
@@ -44,7 +44,7 @@
                           = f.collection_select :industry_id, Industry.order(:name),:id,:name ,{include_blank: t('others.public_words.please_choose')}, { class:"form-control g-color-black g-bg-white g-bg-white--focus g-brd-gray-light-v4 g-brd-primary--hover rounded-3 g-px-15"}
                     .mb-4
                       .actions
-                        = f.button t('others.public_words.save_changes'), class:"btn btn-md btn-block u-btn-primary rounded g-py-13 gb-btn-default text-white"
+                        = f.button t('others.public_words.update'), class:"btn btn-md btn-block u-btn-primary rounded g-py-13 font-weight-bold text-white"
```
Comment (ssharba) [inline app/views/businesses/registrations/edit.html.haml]: @NoorAlkattan we create the class gb-btn-default for our application's buttons. why you override those changes?

Comment (NoorAlkattan) [reply app/views/businesses/registrations/edit.html.haml]: ok

File: app/views/customer_infos/_survey_datatable.html.haml [hunk 2]
```diff
@@ -29,12 +29,12 @@
   .modal-dialog.modal-lg
     / Modal content
     .modal-content
-      .modal-header.bg-secondary.text-white
+      .modal-header.u-btn-primary.font-weight-bold.text-white
```
Comment (ssharba) [inline app/views/customer_infos/_survey_datatable.html.haml]: why you removed bg-secondary class?

Comment (NoorAlkattan) [reply app/views/customer_infos/_survey_datatable.html.haml]: to make the header orange like other style like button style color

Comment (NoorAlkattan) [reply app/views/customer_infos/_survey_datatable.html.haml]: do you want me to put back as bg-secondary?

**Pull request merged**